src/ts-mockito.ts

```diff
@@ -81,6 +81,9 @@ export function cmock<R, T extends any[]>(): new (...args: T) => R {
 }
 
 export function verify<T>(method: T): MethodStubVerificator<T> {
+    if ('expect' in globalThis && 'nothing' in expect()) {
+        expect().nothing();
+    }
```

Comment:
It might be better to set a global boolean, instead of evaluating the full expression on each call to `verify`.